packages/configuration/src/lib.rs

```diff
@@ -804,7 +798,7 @@ mod tests {
     fn http_api_configuration_should_check_if_it_contains_a_token() {
         let configuration = Configuration::default();
 
-        assert!(configuration.http_api.contains_token("MyAccessToken"));
-        assert!(!configuration.http_api.contains_token("NonExistingToken"));
+        assert!(configuration.http_api.access_tokens.values().any(|t| t == "MyAccessToken"));
+        assert!(!configuration.http_api.access_tokens.values().any(|t| t == "NonExistingToken"));
```

Comment:
Hi @da2ce7 I think I would keep the `contains` abstraction:

```rust
pub struct AccessTokens(HashMap<String, String>)

impl AccessTokens {
    /// Checks if the given token is one of the tokens.
    #[must_use]
    pub fn contains(&self, token: &str) -> bool {
        self.0.values().any(|t| t == "MyAccessToken")
    }
}

// ...

assert!(configuration.http_api.access_tokens.contains("MyAccessToken"));
assert!(!configuration.http_api.access_tokens.contains("NonExistingToken"));

```

But It's not important. Maybe it's because I don't know the HashMap API or because I think it's easy to read or because it's against the [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter).

Reply:
@josecelano this is using the standard iterator api, I don't think that it is surprising for any map to provide an iterator.